examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -358,6 +358,16 @@
     [[MTRDeviceControllerFactory sharedInstance] stopControllerFactory];
 }
 
+void CHIPCommandBridge::SuspendOrResumeCommissioners()
+{
+    for (auto & pair : mControllers) {
+        __auto_type * commissioner = pair.second;
+        if (commissioner.running) {
+            commissioner.suspended ? [commissioner resume] : [commissioner suspend];
```

Comment:
Would be nice to log whether we are resuming or suspending (and what the commissioner involved is).

Reply:
The framework is already quite verbose. ;)